.env.example

```diff
@@ -1,7 +1,15 @@
-DB_USERNAME=postgres
-DB_PASSWORD=db_password
-DB_NAME=barefoot_nomad
-DB_HOST=127.0.0.1
-PORT=
-
-DB_TEST_NAME=barefoot_test
+DB_USERNAME=postgres
+DB_PASSWORD=db_password
+DB_NAME=barefoot_nomad
+DB_HOST=127.0.0.1
+PORT=""
+DB_TEST_NAME=barefoot_test
+TOKENEXPIRY=18000
+EMAIL=
+CLIENT_URL="https://firestar_backend.herokuapp.com"
+HOST="localhost"
+DATABASE_URL_TEST=""
+NODE_ENV="test"
+YOUR_EMAIL=""
+SOME_PASSWORD=""

```

Comment:
what is the purpose of the `SOME_PASSWORD`?

Reply:
ditto >> https://github.com/andela/firestar-backend/pull/25#discussion_r318460567